<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CHelperNeoClass</class>
 <widget class="QMainWindow" name="CHelperNeoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CHelperNeo</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_main">
      <item>
       <widget class="QPushButton" name="pushButton_auto">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>55</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="toolTipDuration">
         <number>0</number>
        </property>
        <property name="statusTip">
         <string>红蜘蛛选择窗口化；江波ECR选择解除控制</string>
        </property>
        <property name="text">
         <string>自动选择[解除/启用]控制操作</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_rs">
        <item>
         <widget class="QPushButton" name="pushButton_rs_window">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="toolTipDuration">
           <number>-1</number>
          </property>
          <property name="statusTip">
           <string>窗口化(详见帮助-帮助)</string>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>窗口化</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_rs_full">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="toolTipDuration">
           <number>-1</number>
          </property>
          <property name="statusTip">
           <string>解除窗口化(详见帮助-帮助)</string>
          </property>
          <property name="text">
           <string>恢复全屏</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_rs_dectl">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="toolTipDuration">
           <number>-1</number>
          </property>
          <property name="statusTip">
           <string>解除红蜘蛛控制(详见帮助-帮助)</string>
          </property>
          <property name="text">
           <string>解除控制</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_rs_back">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="toolTipDuration">
           <number>-1</number>
          </property>
          <property name="statusTip">
           <string>关闭菜单</string>
          </property>
          <property name="text">
           <string>返回</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_rs">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="toolTipDuration">
         <number>0</number>
        </property>
        <property name="statusTip">
         <string>进入红蜘蛛菜单</string>
        </property>
        <property name="text">
         <string>红蜘蛛 菜单</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_jp">
        <item>
         <widget class="QPushButton" name="pushButton_jp_patch">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="statusTip">
           <string>安装后可以在被控制的情况下进行其他操作(详见帮助-帮助)</string>
          </property>
          <property name="text">
           <string>安装补丁</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_jp_dectl">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="statusTip">
           <string>解除江波ECR控制(详见帮助-帮助)</string>
          </property>
          <property name="text">
           <string>解除控制</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_jp_back">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="statusTip">
           <string>关闭菜单</string>
          </property>
          <property name="text">
           <string>返回</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_jp">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="toolTipDuration">
         <number>0</number>
        </property>
        <property name="statusTip">
         <string>进入江波ECR菜单</string>
        </property>
        <property name="text">
         <string>江波ECR 菜单</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_rectl">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTipDuration">
         <number>0</number>
        </property>
        <property name="statusTip">
         <string>恢复使用了解除控制的软件的控制</string>
        </property>
        <property name="text">
         <string>开启控制</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_patch">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="statusTip">
         <string>安装后在控制中按五下 Shift 可解除控制。</string>
        </property>
        <property name="text">
         <string>安装增强补丁</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_stat">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>红蜘蛛：江波ECR：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_patch">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>补丁状态：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>340</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_main">
    <property name="statusTip">
     <string>查看所有可用操作</string>
    </property>
    <property name="title">
     <string>操作</string>
    </property>
    <widget class="QMenu" name="menu_rs">
     <property name="statusTip">
      <string>红蜘蛛相关操作</string>
     </property>
     <property name="title">
      <string>红蜘蛛</string>
     </property>
     <addaction name="action_rs_window"/>
     <addaction name="action_rs_dectl"/>
     <addaction name="separator"/>
     <addaction name="action_rs_full"/>
     <addaction name="action_rs_rectl"/>
    </widget>
    <widget class="QMenu" name="menu_jp">
     <property name="statusTip">
      <string>江波ECR相关操作</string>
     </property>
     <property name="title">
      <string>江波ECR</string>
     </property>
     <addaction name="action_jp_patch"/>
     <addaction name="action_jp_dectl"/>
     <addaction name="separator"/>
     <addaction name="action_jp_rectl"/>
    </widget>
    <addaction name="menu_rs"/>
    <addaction name="menu_jp"/>
    <addaction name="separator"/>
    <addaction name="action_rop"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="statusTip">
     <string>帮助</string>
    </property>
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_help"/>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu_main"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_quit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="statusTip">
    <string>退出程序</string>
   </property>
  </action>
  <action name="action_rs_window">
   <property name="text">
    <string>窗口化</string>
   </property>
   <property name="toolTip">
    <string>窗口化</string>
   </property>
   <property name="statusTip">
    <string>窗口化(详见帮助-帮助)</string>
   </property>
  </action>
  <action name="action_rs_dectl">
   <property name="text">
    <string>解除控制</string>
   </property>
   <property name="toolTip">
    <string>解除控制</string>
   </property>
   <property name="statusTip">
    <string>解除红蜘蛛控制(详见帮助-帮助)</string>
   </property>
  </action>
  <action name="action_rs_rectl">
   <property name="text">
    <string>开启控制</string>
   </property>
   <property name="toolTip">
    <string>开启控制</string>
   </property>
   <property name="statusTip">
    <string>恢复使用了解除控制的红蜘蛛的控制</string>
   </property>
  </action>
  <action name="action_jp_rectl">
   <property name="text">
    <string>开启控制</string>
   </property>
   <property name="statusTip">
    <string>恢复江波ECR控制</string>
   </property>
  </action>
  <action name="action_jp_dectl">
   <property name="text">
    <string>解除控制</string>
   </property>
   <property name="statusTip">
    <string>解除江波ECR控制(详见帮助-帮助)</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>帮助</string>
   </property>
   <property name="statusTip">
    <string>帮助</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
   <property name="statusTip">
    <string>关于</string>
   </property>
  </action>
  <action name="action_rs_full">
   <property name="text">
    <string>恢复全屏</string>
   </property>
   <property name="toolTip">
    <string>恢复全屏</string>
   </property>
   <property name="statusTip">
    <string>解除窗口化(详见帮助-帮助)</string>
   </property>
  </action>
  <action name="action_rop">
   <property name="text">
    <string>右键加速</string>
   </property>
   <property name="statusTip">
    <string>解决点击右键卡顿问题</string>
   </property>
  </action>
  <action name="action_jp_patch">
   <property name="text">
    <string>安装补丁</string>
   </property>
   <property name="statusTip">
    <string>安装后可以在被控制的情况下进行其他操作(详见帮助-帮助)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
